packages/ui/src/components/va-color-indicator/VaColorIndicator.vue

```diff
@@ -79,17 +79,14 @@ export default defineComponent({
     border-color: var(--va-primary);
   }
 
-  &--hoverable &__core:hover,
-  &:focus {
-    transform: scale(1.1);
-    transition: transform 0.1s linear;
-  }
-
   &__core {
-    transition: transform 0.1s linear;
-    border-radius: 50%;
-    width: 1rem;
+    border-radius: v-bind(borderRadiusComputed);
     height: 1rem;
+    width: 1rem;
+  }
+
+  &:focus {
```

Comment:
Not `keyboard-only-focus` mixin? Not sure if we want outline for mouse interactions.